### ISSUE TITLE
Does fabric3 support running the watch command remotely?

### ISSUE THREAD

**Issue by zhangjian94cn** [fabric/fabric#2270]
For example, I want to run `watch -n 1 nvidia-smi` command using fabric3 and print the result in local machine. How to realize it?

Thank you very much~

**Comment by davidjmemmett** [comment]
Yes, please read the [Getting started documentation](https://docs.fabfile.org/en/latest/getting-started.html).
